### ISSUE TITLE
THE MACHINE THE TERROR MACHINE THE LAG MACHINE

### ISSUE THREAD

**Issue by SethLafuente** [tgstation/tgstation#71222]
## Round ID:

A with a single click can end entire servers everyone must fear its existence and it must be dealt with

In One I think this should be a feature but thats up to the head admins right now I guess and coders this was definitely intended right ;)

## Testmerges:
![image](https://user-images.githubusercontent.com/84478872/201456494-e49611a0-c5b0-40be-9635-849eceb48b45.png)

## Reproduction:
Basically unscrewdrive more than 100 igniters the more the more lag it does when used it could even make the server crash with a single click and attach the igniters together making it all at once do an effect that makes shit lag bad
